go/tools/builders/stdlib.go

```diff
@@ -24,13 +24,14 @@ import (
 )
 
 func install_stdlib(goenv *GoEnv, target string, args []string) error {
+	if goenv.tags != "" {
+		args = append(args, "-tags", goenv.tags)
```

Comment:
You mentioned `ios` in the description. I wonder if there's a way we can set that one automatically?

Reply:
At the moment the only way I found to reliably do it is by using `ctx.fragments.cpp.target_gnu_system_name`.

Here is the gist of the code, more or less (I plan to submit this later on):
```python
PLATFORMS = {
    'armv7-apple-ios': apple_common.platform.ios_device,
    'arm64-apple-ios': apple_common.platform.ios_device,
    'i386-apple-ios': apple_common.platform.ios_simulator,
    'x86_64-apple-ios': apple_common.platform.ios_simulator,
}

def apple_ensure_options(ctx, env, tags, compiler_options, linker_options):
    system_name = ctx.fragments.cpp.target_gnu_system_name
    platform = PLATFORMS.get(system_name)
    if platform == None:
        return
    if system_name.endswith("-ios"):
        tags.append("ios") # needed for stdlib building
    if platform in [apple_common.platform.ios_device, apple_common.platform.ios_simulator]:
        min_version = _apple_version_min(platform, "6.1")
        compiler_options.append(min_version)
        linker_options.append(min_version)
    xcode_config = ctx.attr._xcode_config[apple_common.XcodeVersionConfig]
    env.update(apple_common.target_apple_env(xcode_config, platform))
```

Reply:
As you can see also, this method modifies the environment, and in this case to tell the compiler which SDK to use (iPhoneOS, iPhoneSimulator).

For instance:
```
    APPLE_SDK_PLATFORM=iPhoneOS
    APPLE_SDK_VERSION_OVERRIDE=11.2
    XCODE_VERSION_OVERRIDE=9.2.0
```

These also need to be set at stdlib bootstrapping.